### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
🏋️‍♀️ related https://github.com/paritytech/polkadot/pull/6025

### CODE REVIEW

File: runtime/kusama/src/weights/frame_election_provider_support.rs [hunk 1]
```diff
@@ -48,20 +48,20 @@ impl<T: frame_system::Config> frame_election_provider_support::WeightInfo for We
...
 	/// The range of component `d` is `[5, 16]`.
 	fn phragmms(v: u32, _t: u32, d: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 79_000
-			.saturating_add(Weight::from_ref_time(14_480_000 as u64).saturating_mul(v as u64))
-			// Standard Error: 6_844_000
-			.saturating_add(Weight::from_ref_time(2_525_332_000 as u64).saturating_mul(d as u64))
+		Weight::from_ref_time(4_332_183_000 as u64)
+			// Standard Error: 146_563
+			.saturating_add(Weight::from_ref_time(5_508_430 as u64).saturating_mul(v as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_election_provider_support.rs:63]: 0 -> 4b

File: runtime/kusama/src/weights/frame_system.rs [hunk 2]
```diff
@@ -38,53 +38,56 @@
...
 /// Weight functions for `frame_system`.
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> frame_system::WeightInfo for WeightInfo<T> {
 	/// The range of component `b` is `[0, 3932160]`.
 	fn remark(b: u32, ) -> Weight {
-		Weight::from_ref_time(882_000 as u64)
+		Weight::from_ref_time(3_031_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(1_000 as u64).saturating_mul(b as u64))
+			.saturating_add(Weight::from_ref_time(512 as u64).saturating_mul(b as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_system.rs:51]: 800k -> 3m

File: runtime/kusama/src/weights/frame_system.rs [hunk 3]
```diff
@@ -38,53 +38,56 @@
...
 	}
 	/// The range of component `b` is `[0, 3932160]`.
 	fn remark_with_event(b: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
+		Weight::from_ref_time(11_653_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(2_000 as u64).saturating_mul(b as u64))
+			.saturating_add(Weight::from_ref_time(1_829 as u64).saturating_mul(b as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_system.rs:57]: 0 -> 11m

File: runtime/kusama/src/weights/frame_system.rs [hunk 4]
```diff
@@ -38,53 +38,56 @@
...
 	}
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `i` is `[1, 1000]`.
 	fn set_storage(i: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 2_000
-			.saturating_add(Weight::from_ref_time(610_000 as u64).saturating_mul(i as u64))
+		Weight::from_ref_time(3_843_000 as u64)
+			// Standard Error: 1_590
+			.saturating_add(Weight::from_ref_time(578_944 as u64).saturating_mul(i as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_system.rs:71]: 0 -> 3.8m

File: runtime/kusama/src/weights/frame_system.rs [hunk 5]
```diff
@@ -38,53 +38,56 @@
...
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `i` is `[1, 1000]`.
 	fn kill_storage(i: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 1_000
-			.saturating_add(Weight::from_ref_time(454_000 as u64).saturating_mul(i as u64))
+		Weight::from_ref_time(3_896_000 as u64)
+			// Standard Error: 955
+			.saturating_add(Weight::from_ref_time(434_545 as u64).saturating_mul(i as u64))
+			.saturating_add(T::DbWeight::get().writes(1 as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_system.rs:81]: 0 -> 3.8m

File: runtime/kusama/src/weights/frame_system.rs [hunk 6]
```diff
@@ -38,53 +38,56 @@
...
 	// Storage: Skipped Metadata (r:0 w:0)
 	/// The range of component `p` is `[1, 1000]`.
 	fn kill_prefix(p: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 1_000
-			.saturating_add(Weight::from_ref_time(978_000 as u64).saturating_mul(p as u64))
+		Weight::from_ref_time(6_337_000 as u64)
+			// Standard Error: 1_174
+			.saturating_add(Weight::from_ref_time(937_289 as u64).saturating_mul(p as u64))
+			.saturating_add(T::DbWeight::get().writes(1 as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/frame_system.rs:90]: 0  -> 6m

File: runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs [hunk 7]
```diff
@@ -139,16 +135,10 @@ impl<T: frame_system::Config> pallet_election_provider_multi_phase::WeightInfo f
...
-			// Standard Error: 14_000
-			.saturating_add(Weight::from_ref_time(39_000 as u64).saturating_mul(t as u64))
-			// Standard Error: 23_000
-			.saturating_add(Weight::from_ref_time(10_873_000 as u64).saturating_mul(a as u64))
-			// Standard Error: 35_000
-			.saturating_add(Weight::from_ref_time(2_257_000 as u64).saturating_mul(d as u64))
+	fn submit_unsigned(_v: u32, _t: u32, a: u32, _d: u32, ) -> Weight {
+		Weight::from_ref_time(6_892_478_000 as u64)
+			// Standard Error: 57_196
+			.saturating_add(Weight::from_ref_time(6_883_886 as u64).saturating_mul(a as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/pallet_election_provider_multi_phase.rs:141]: 0   > 7b no longer using params ?

File: runtime/kusama/src/weights/pallet_elections_phragmen.rs [hunk 8]
```diff
@@ -149,15 +149,14 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
...
-			.saturating_add(T::DbWeight::get().reads((3 as u64).saturating_mul(v as u64)))
-			.saturating_add(T::DbWeight::get().writes((3 as u64).saturating_mul(v as u64)))
+	fn clean_defunct_voters(v: u32, _d: u32, ) -> Weight {
+		Weight::from_ref_time(282_352_767_000 as u64)
+			// Standard Error: 243_946
+			.saturating_add(Weight::from_ref_time(35_271_908 as u64).saturating_mul(v as u64))
+			.saturating_add(T::DbWeight::get().reads(15004 as u64))
+			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(v as u64)))
+			.saturating_add(T::DbWeight::get().writes(15000 as u64))
+			.saturating_add(T::DbWeight::get().writes((2 as u64).saturating_mul(v as u64)))
```
Comment (chevdor) [inline runtime/kusama/src/weights/pallet_elections_phragmen.rs:159]: 0 > 280b

File: runtime/kusama/src/weights/pallet_elections_phragmen.rs [hunk 9]
```diff
@@ -172,14 +171,14 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
...
-			// Standard Error: 46_000
-			.saturating_add(Weight::from_ref_time(3_139_000 as u64).saturating_mul(e as u64))
-			.saturating_add(T::DbWeight::get().reads(265 as u64))
-			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(c as u64)))
+		Weight::from_ref_time(29_157_699_000 as u64)
+			// Standard Error: 525_734
+			.saturating_add(Weight::from_ref_time(43_642_232 as u64).saturating_mul(v as u64))
+			// Standard Error: 33_738
+			.saturating_add(Weight::from_ref_time(2_196_265 as u64).saturating_mul(e as u64))
+			.saturating_add(T::DbWeight::get().reads(1968 as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/pallet_elections_phragmen.rs:179]: 0 > 29b

File: runtime/kusama/src/weights/pallet_preimage.rs [hunk 10]
```diff
@@ -48,87 +48,87 @@ impl<T: frame_system::Config> pallet_preimage::WeightInfo for WeightInfo<T> {
 	// Storage: Preimage StatusFor (r:1 w:1)
 	/// The range of component `s` is `[0, 4194304]`.
 	fn note_preimage(s: u32, ) -> Weight {
-		Weight::from_ref_time(0 as u64)
-			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(2_000 as u64).saturating_mul(s as u64))
+		Weight::from_ref_time(30_131_000 as u64)
+			// Standard Error: 1
+			.saturating_add(Weight::from_ref_time(2_441 as u64).saturating_mul(s as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/pallet_preimage.rs:53]: 0 > 30m

File: runtime/kusama/src/weights/pallet_staking.rs [hunk 11]
```diff
@@ -364,29 +363,26 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
...
-			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(s as u64)))
+	fn get_npos_voters(v: u32, n: u32, _s: u32, ) -> Weight {
+		Weight::from_ref_time(25_645_050_000 as u64)
+			// Standard Error: 467_573
+			.saturating_add(Weight::from_ref_time(10_912_850 as u64).saturating_mul(v as u64))
+			// Standard Error: 467_573
+			.saturating_add(Weight::from_ref_time(10_774_564 as u64).saturating_mul(n as u64))
+			.saturating_add(T::DbWeight::get().reads(6706 as u64))
+			.saturating_add(T::DbWeight::get().reads((2 as u64).saturating_mul(v as u64)))
+			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(n as u64)))
```
Comment (chevdor) [inline runtime/kusama/src/weights/pallet_staking.rs:374]: 0 > 25b

File: runtime/kusama/src/weights/runtime_parachains_paras_inherent.rs [hunk 12]
```diff
@@ -148,10 +151,10 @@ impl<T: frame_system::Config> runtime_parachains::paras_inherent::WeightInfo for
...
 	/// The range of component `v` is `[101, 200]`.
 	fn enter_backed_candidates_variable(v: u32, ) -> Weight {
-		Weight::from_ref_time(907_441_000 as u64)
-			// Standard Error: 47_000
-			.saturating_add(Weight::from_ref_time(48_158_000 as u64).saturating_mul(v as u64))
-			.saturating_add(T::DbWeight::get().reads(28 as u64))
+		Weight::from_ref_time(5_768_526_000 as u64)
+			// Standard Error: 268_593
+			.saturating_add(Weight::from_ref_time(17_016_045 as u64).saturating_mul(v as u64))
+			.saturating_add(T::DbWeight::get().reads(29 as u64))
```
Comment (chevdor) [inline runtime/kusama/src/weights/runtime_parachains_paras_inherent.rs:157]: 900m > 5b

**Review by chevdor — CHANGES_REQUESTED** [review]
Reviewing the values, this run seems rather odd compared to the several previous releases we had.

The weights in some pallets look great (= very steady as we are used to). For others however, we can see huge bumps and quite some from 0 to several billions.

@coderobe  we should start a new benchmark run. We can decide later to use the results or not.

I also would like to gather @kianenigma & @ggwpez's opinions on the big changes I spotted. There are more bing changes than whan I reported.

**Comment by ggwpez** [comment]
A lot of the changes are probably due to https://github.com/paritytech/substrate/pull/11806  
I still wonder why for example the components here are now unused:  
image


cc @koute @shawntabrizi

**Comment by coderobe** [comment]
not merging this round

**Pull request closed without merging**